kepub/converter.go

```diff
@@ -83,6 +88,43 @@ func (c *Converter) Convert(dir string) error {
 	return nil
 }
 
+func (c *Converter) GetHtmlFiles(dir string) ([]string, error) {
+	opfPath, err := FindOPF(dir)
+	if err != nil {
+		return []string{}, fmt.Errorf("transform opf: find: %w", err)
+	}
+
+	rsk, err := os.Open(opfPath)
+	if err != nil {
+		return []string{}, fmt.Errorf("error opening opf file: %w", err)
+	}
+	defer rsk.Close()
+
+	doc := etree.NewDocument()
+	if _, err = doc.ReadFrom(rsk); err != nil {
+		return []string{}, fmt.Errorf("error parsing opf file: %w", err)
+	}
+
+	var files []string
+
+	elements := doc.FindElements("//package/manifest/item")
```

Comment:
I tried to use the syntax below. But it didn't work.

doc.FindElements("//package/manifest/item[contains(@media-type,'text/html')]")
doc.FindElements("//package/manifest/item[contains(@media-type,'application/xhtml+xml')]")

Reply:
Yes, that's the expected behaviour for [etree](https://github.com/beevik/etree/blob/4a2f8b9d084c5fffa2fe44a73bd8efaf7dcda53a/path.go#L12-L83). It only implements a very small subset of XPath.